scripts/container.sh

```diff
@@ -7,6 +7,7 @@ cd "$(dirname "$0")/.."
 
 # normalize branch name to reflect a valid image name
 BRANCH=$(git branch --show-current | sed 's/[^a-z0-9-]/_/ig')
+[ -z "${BRANCH}" ] && BRANCH=default
 TAG=gluon:${BRANCH}
```

Comment:
I think using `latest` as the fallback makes more sense. This can also be written more concisely:
```suggestion
TAG="gluon:${BRANCH:-latest}"
```

Reply:
nice, I googled for a short oneliner like this but hadn't found anything useful.